js/components/sharePanel.js

```diff
@@ -96,16 +96,23 @@ var SharePanel = React.createClass({
     mailToUrl += "?subject=" + encodeURIComponent(this.props.contentTree.title);
     mailToUrl += "&body=" + encodeURIComponent(emailBody + location.href);
     //location.href = mailToUrl; //same window
-    var emailWindow = window.open(mailToUrl, "email", "height=315,width=780"); //new window
-    setTimeout(function(){
-      try {
-         // If we can't access href, a web client has taken over and this will throw
-         // an exception, preventing the window from being closed.
-        var test = emailWindow.location.href;
-        emailWindow.close()
-      } catch(e) {};
-      // Generous 2 second timeout to give the window time to redirect if it's going to a web client
-    }, 2000);
+    if (OO.isIos && OO.isSafari) {
+        document.location = mailToUrl;
+    } else {
+        var emailWindow = window.open(mailToUrl, "email", "height=315,width=780"); //new window
+        setTimeout(function () {
+            try {
```

Comment:
The style in this PR is inconsistent with the rest of the code. The indentation in the rest of the file uses 2 spaces.

Reply:
Okay. I will correct.

Reply:
@jpfranco I have corrected the indentation and unwanted semicolon.

Please approve the PR